easybuild/easyconfigs/n/NCIPLOT/NCIPLOT-4.0-20190718-iccifort-2019.5.281.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'MakeCp'
+
+name = 'NCIPLOT'
+local_ver = '4.0'
+version = '%s-20190718' % local_ver
+local_commit = '834af2e'
+
+homepage = 'https://www.lct.jussieu.fr/pagesperso/contrera/index-nci.html'
+description = """ NCIPLOT is a program for revealing non covalent interactions
+ based on the reduced density gradient. """
+
+toolchain = {'name': 'iccifort', 'version': '2019.5.281'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/juliacontrerasgarcia/nciplot/archive/']
+sources = ['%s.tar.gz' % local_commit]
+checksums = ['d86943e7f4dff9098b87fdfb8ab3793478a9c8ed8b312c730cdbcb0cac4bef4d']
+
+start_dir = 'src_%%(namelower)s_%s' % local_ver
+
+prebuildopts = "sed -i 's/include Makefile.inc//g' Makefile && "
+buildopts = 'LDFLAGS="$LDFLAGS $LIBS"'
+
+files_to_copy = ['dat', 'LICENSE', (['nciplot'], 'bin')]
+
+modextrapaths = {'NCIPLOT_HOME': ''}
+
+sanity_check_paths = {
+    'files': ['bin/nciplot'],
+    'dirs': ['dat'],
+}
+
+# site-specific: set OMP_NUM_THREADS equal to number of available cores
+modluafooter = 'if os.getenv("PBS_NUM_PPN") ~= nil then pushenv("OMP_NUM_THREADS", os.getenv("PBS_NUM_PPN")) end'
```

Comment:
This part doesn't really belong here. It's the responsibility of the user to set OMP_NUM_THREADS.
(And the responsibility of the site documentation to tell them how and when :-)